clang/lib/3C/3C.cpp

```diff
@@ -502,11 +510,12 @@ bool _3CInterface::writeConvertedFileToDisk(const std::string &FilePath) {
     if (RewriteTool) {
       int ToolExitCode = Tool.run(RewriteTool.get());
       if (ToolExitCode != 0)
-        return false;
+        RetVal = false;
     }
-    return true;
   }
-  return false;
+  GlobalProgramInfo.getPerfStats().endTotalTime();
+  GlobalProgramInfo.getPerfStats().startTotalTime();
```

Comment:
This looks weird. You end and then start total time in several places; why? I would have thought these calls book-end the whole process.

Reply:
This is to keep the time updated at each step. `endTotalTime` will update the total time and `startTotalTime` will restart the timer.